api/src/config/mod.rs

```diff
@@ -196,9 +196,7 @@ impl Config {
     }
 
     pub fn hook(&self, hook_name: &str) -> Option<&PathBuf> {
-        self.hooks
-            .as_ref()
-            .and_then(|hooks| hooks.get(hook_name))
+        self.hooks.as_ref().and_then(|hooks| hooks.get(hook_name))
```

Comment:
That's a change that is not related to PR's contribution. Please, revert.

Reply:
This has been reverted.